backup_and_restore/application_backup_and_restore/oadp-performance/oadp-recommended-network-settings.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="oadp-recommended-network-settings.adoc"]
```

Comment:
@anarnold97, the ".adoc" part of this ID is making the PV1 builds fail. Will you open a new PR to remove it?

Reply:
@kalexand-rh  PTAL https://github.com/openshift/openshift-docs/pull/84386

Reply:
@kalexand-rh - https://github.com/openshift/openshift-docs/pull/84387